src/public/js/script.js

```diff
@@ -115,25 +117,32 @@ async function postData() {
       quantity3: qty3Input,
       powerOfUnits: powerUnitsInput,
       additionalNote: additionalNotesInput,
-      fromCity1: fromCity1Inputs,
-      fromCity2: fromCity2Inputs,
-      fromCity3: fromCity3Inputs,
-      fromCity4: fromCity4Inputs,
-      fromCity5: fromCity5Inputs,
-      toCity1: toCity1Inputs,
-      toCity2: toCity2Inputs,
-      toCity3: toCity3Inputs,
-      toCity4: toCity4Inputs,
-      toCity5: toCity5Inputs,
       optionCity: optionCityInput,
       optionState: optionStateInput,
       optionZip: optionZipInput,
     })
-    showAlert('success', 'Form submitted successfully!')
+    return await response
   } catch (error) {
-   throw new Error()
+    throw new Error()
   }
 }
+async function addDesiredLanes(){
+  const fromCityOptionalInput = document.getElementById('fromCity').value
+  const toCityOptionalInput = document.getElementById('toCity').value
+  try {
+    const response = await axios.post('http://localhost:80/api/desiredLanes', [{
+      fromCity : fromCityOptionalInput,
+      toCity : toCityOptionalInput,
+      customerInfoUuid : uniqueUuid,
+    }])
+    return await response
```

Comment:
This cannot not work, in array it always have one element

Reply:
Relation should not be decided from where, btw will not work

Reply:
done